sagemaker-script-mode/pytorch_bert/deploy_bert.ipynb

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
<p>We should switch delete_endpoint and delete_model as that has caused error in the past before. But it is interesting that CI still passed with this ordering - it used to fail</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Switched order to delete the model first, then endpoint.